api/turing/api/ensemblers_api.go

```diff
@@ -149,14 +161,20 @@ func (c EnsemblersController) UpdateEnsembler(
 		}
 	}
 
+	// call webhook for ensembler update event
+	if errWebhook := c.webhookClient.TriggerEnsemblerEvent(ctx, webhook.OnEnsemblerUpdated, ensembler); errWebhook != nil {
```

Comment:
what happens if the trigger is failing but the operation is successful?

Reply:
Webhook will retry to send the event because the mechanism already implemented in MLP package https://github.com/caraml-dev/mlp/blob/9b920121bf6af8bab1922efdaaa8c0411afbd522/api/pkg/webhooks/client.go#L81-L110